internal/opengl/opengl.go

```diff
@@ -0,0 +1,77 @@
+// +build !openglnone
+
+package opengl
+
+// The default version (3.3) of OpenGL used by go-flutter.
+// If you want to support other version, copy/pase this file, change the import
+// statement, add builds constraints and open a PR.
+
+import (
+	"unsafe"
+
+	"github.com/go-gl/gl/v3.3-core/gl"
+	"github.com/go-gl/glfw/v3.2/glfw"
```

Comment:
3.2? Not 3.3?

Reply:
I've chosen 3.3 was based on [this](https://www.reddit.com/r/opengl/comments/51htbk/what_is_the_best_opengl_version_to_start_with/d7c7erf?utm_source=share&utm_medium=web2x) comment from a Reddit post.

> Short answer: a core OpenGL 3.3 profile. You'll get wide support across all major operating systems with this.
> OS X and Mesa will be your lowest-end target APIs, and they bottom out at GL 3.3 for the oldest and gimpiest. GL 3.3 is a "modern" API meaning that a lot of what you do here will be comparable to other 3D APIs in concept and execution.


The post has 3years but I think it is still relevant as OpenGL moves very slowly.

I'm NOT familiar with OpenGL, if you think 3.2 is a better choice, I will downgrade. (Not comfortable enough to have any opinion)